# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../pciutils/libpci.patch
# Copyright (C) 2006 - 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2007-05-04 05:43:16.000000000 -0400
+++ ./Makefile	2007-05-04 16:59:08.000000000 -0400
@@ -9,6 +9,8 @@
 
 PREFIX=/usr/local
 SBINDIR=$(PREFIX)/sbin
+LIBDIR=$(PREFIX)/lib
+INCDIR=$(PREFIX)/include
 SHAREDIR=$(PREFIX)/share
 IDSDIR=$(SHAREDIR)
 MANDIR:=$(shell if [ -d $(PREFIX)/share/man ] ; then echo $(PREFIX)/share/man ; else echo $(PREFIX)/man ; fi)
@@ -62,6 +64,9 @@
 	$(INSTALL) -c -m 755 update-pciids $(DESTDIR)$(SBINDIR)
 	$(INSTALL) -c -m 644 $(PCI_IDS) $(DESTDIR)$(IDSDIR)
 	$(INSTALL) -c -m 644 lspci.8 setpci.8 update-pciids.8 $(DESTDIR)$(MANDIR)/man8
+	$(INSTALL) -c -m 755 -d $(DESTDIR)$(INCDIR)/pci
+	$(INSTALL) -c -m 644 lib/libpci.a $(DESTDIR)$(LIBDIR)/
+	$(INSTALL) -c -m 644 lib/*.h $(DESTDIR)$(INCDIR)/pci/
 
 uninstall: all
 	rm -f $(DESTDIR)$(SBINDIR)/lspci $(DESTDIR)$(SBINDIR)/setpci $(DESTDIR)$(SBINDIR)/update-pciids
